ports/stm32f4xx/dlmstp.c

```diff
@@ -442,6 +448,11 @@ static void MSTP_Receive_Frame_FSM(void)
                     /* Preamble1 */
                     /* receive the remainder of the frame. */
                     Receive_State = MSTP_RECEIVE_STATE_PREAMBLE;
+
+                    /* if a frame-start callback was provided, call it */
+                    if (PreamCallback != NULL) {
+                      PreamCallback();
```

Comment:
please run [clang-format](https://llvm.org/builds/) on these code changes, or just the new code before submitting so that the style matches the existing style (in the root .clang-format file).

Reply:
Updated